resources/mac/helpers/org.tidepool.disk-permissions/install.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+BASE_PATH="$(cd "$(dirname "${0}")"; pwd)"
```

Comment:
This would set `BASE_PATH` to `pwd` even if the `cd` failed. To mitigate this, you can make it use `&&` instead of `;` (which would make `BASE_PATH` be `""`). You could then validate that the `cd` was valid by checking `-n $BASE_PATH` (or throw an error if `-z $BASE_PATH`).
It may also not hurt to add to redirect STDERR to /dev/null for the `cd` so that the error doesn't print out (since this script executes no matter what anyway).
So, in it's entirety:
```
BASE_PATH="$(cd "$(dirname "${0}")" 2>/dev/null && pwd)"
if [ -z $BASE_PATH ]; then
   echo "Well, that went poorly"
   exit 1
fi
```

Reply:
Should that be `test -z` ?